test/onnx/gen_onnx.py

```diff
@@ -1016,6 +1016,51 @@ def deconv_stride_test():
     return ([node], [x, w], [y])
 
 
+@onnx_test
+def depthtospace_test():
+
+    x = helper.make_tensor_value_info('x', TensorProto.FLOAT, [2, 8, 5, 5])
```

Comment:
Maybe use dtype other than float for one of the tests ?

Reply:
Not following, what's the benefit?

Reply:
No benefit, but just  to assert it works with other dtypes as well. Up to you if you want to make change.